src/components/minter/mint.cairo

```diff
@@ -259,7 +258,7 @@ mod MintComponent {
             let isOwner = project.only_owner(caller_address);
             assert(isOwner, 'Caller is not the owner');
 
-            let max_money_amount_per_tx = self.get_max_money_amount();
+            let max_money_amount_per_tx = project.get_max_money_amount();
```

Comment:
Should we use get_project_carbon here?
The idea is that the project should already know about project_carbon and care less about "money".
- Project knows about carbon amounts
- Minter knows about carbon to money conversions.